challenges.yml

```diff
@@ -0,0 +1,184 @@
+addsimplenamespacecoding:
+    question: >
+        Everyone at the office is *starving*, but you want to
+        master namespaces before celebrating with lunch.
+
+        This `Pizza` class looks great. But now, give the
+        class a namespace - `Food\Tasty`. Then, update `eat.php`
+        to work, but **without** adding a `use` statement.
+
+        Everyone is starving, and depending on you!
+    step: add-the-pizza-class
+    files:
+        eat.php: ~
+        Pizza.php: ~
+    features:
+        editor: ~
+        browser:
+            default_uri: /eat.php
+    grading:
+        check_classes:
+            type: php
+            expect: "(not class_exists('Food\Tasty\Pizza')) and class_exists('Pizza')"
```

Comment:
Done in https://github.com/KnpLabs/KnpUniversity/commit/5a6f2568b2ab8ac7b54e189397b321f0649194d9